### PR TITLE
GEN-637 - feat: added TrustpilotBlock

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Added a `TrustpilotBlock` that displays _Trustpilot_ data.

How the whole thing works:

1. Real data is stored in store's [_Edge Config_ storage](https://vercel.com/hedvig/hedvig-dot-com/stores/edge-config/ecfg_viia3ntdquafelygrc3g6lfs3d2a/items#item=trustpilot);
2. This PR will set a cron job that's going to update it every Monday at 8:00 (just picked that number. Gonna double check if that's enough) - check `apps/store/vercel.json` and `apps/store/src/pages/api/cron/trustpilot.ts` files
3. There's a new service - `apps/store/src/services/trustpilot/trustpilot.tsx` - which is responsible to provide a way to retrieve that data from the storage. It also gonna store it in a Jotai's atom so it can be available in the whole app. We then fetch that data and proper hydrate the atom in the `apps/store/src/pages/[[...slug]].tsx` and `apps/store/src/components/LayoutWithMenu/LayoutWithMenu.tsx` files respectively.
4. `TrustpilotBlock` then reads that data from the atom and proper display it.

Screenshot 2023-07-06 at 17 41 54

## Justify why they are needed

Requested by design/editors

Can be tested [here](https://hedvig-dot-com-git-gen-637-feattrustpilot-blocksplit2-hedvig.vercel.app/se/test-home?__vercel_draft=1)

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2705** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2705" target="_blank">Graphite</a> 
    * **PR #2710** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2710" target="_blank">Graphite</a> 
      * **PR #2711** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2711" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2711?utm_source=stack-comment).

**Review by gustaveen — COMMENTED** [review]
Nice work! 
On thing to consider is to put the text and url as fields in Storyblok instead. Then we could make a reusable block of it

**Comment by guilhermespopolin** [comment]
> Nice work! 🙌 On thing to consider is to put the text and url as fields in Storyblok instead. Then we could make a reusable block of it

Have update it so we get the link from Storyblok

File: apps/store/src/blocks/TrustpilotBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.div({
+  display: 'grid',
+  placeItems: 'center',
+  backgroundColor: theme.colors.signalGreenFill,
+  paddingBlock: '10rem',
+  paddingInline: '1rem',
```
Comment (robinandeer) [inline apps/store/src/blocks/TrustpilotBlock.tsx:47]: Why not use theme?

File: apps/store/src/blocks/TrustpilotBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  gap: theme.space.md,
+})
+
+const ScoreText = styled(Text)({
+  lineHeight: '1.32',
+  // Font size tihs high is an edge case. That's why I'm not using Text's 'size' prop
```
Comment (robinandeer) [inline apps/store/src/blocks/TrustpilotBlock.tsx:60]: nit: typo "tihs"

**Review by robinandeer — APPROVED** [review]
Look good!

**Comment by guilhermespopolin** [comment]
@guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2711).

**Pull request merged**